### DISCUSSION TITLE
Automatically view embed list

### DISCUSSION THREAD

**Discussion by shawnpaak** [zsviczian/obsidian-excalidraw-plugin#656] [Q&A] [answered]
Is there a function to automatically view the list that can be embed(md notes or especially excalidraw files!) 
when typing double brackets as seen in the obsidian markdown editor?
Any other way is fine than double brackets.

Please excuse if my explanation was terrible. 
I don't know the exact name of it. Please refer to screenshots below.

<img width="600" alt="2022-06-10_23-56-22" src="https://user-images.githubusercontent.com/103835042/173093734-5295c017-5b3d-4b4a-93ff-e84189a9dd97.png">

<img width="800" alt="2022-06-10_23-59-09" src="https://user-images.githubusercontent.com/103835042/173093789-dfddb2b3-4c35-4d2f-8e35-e00276b4f841.png">

I'd really appreciate any help.
Thank you.

LONG LIVE THE EXCALIDRAW !!

**Comment by zsviczian** [comment, marked as answer]
Unfortunately, there isn't (yet). You have three alternatives:
- I have a command palette action to insert a link, a markdown file, or an image.
- Similarly in Excalidraw on the Obsidian toolbar I have actions for these
- You can drag and drop items from the file explorer

**Reply by shawnpaak** [reply]
Wow, there's already have function that I want. 
I can use them to do exactly what I want to do. It's actually more than that. 
As expected, Excalidraw is an all-powerful tool that can do everything as i imagine. 
Thank you very much.
